index.js

```diff
@@ -10,11 +10,8 @@ module.exports = function(out, options) {
 
   var onFile = function(file) {
     files.push(file);
-    var path = file.path;
-    if (!options.absolute) {
-      path = path.replace(new RegExp('^' + process.cwd() + '/'), '');
-    }
-    filePaths.push(path);
+    var path = options.absolute ? file.path : file.relative;
```

Comment:
I'm a little confused by this change - it's returning `test.file` instead of `test/test.file` which isn't the behavior that I was going for. It should be relative to the currently executing process's directory. Run `npm test` to see what I mean.